gradle/common/dependencies.gradle

```diff
@@ -16,10 +16,10 @@ dependencyManagement {
         }
 
         dependencySet(group: 'org.jetbrains.kotlin', version: kotlinVersion) {
-            entry 'kotlin-stdlib'
+            entry 'kotlin-stdlib-common'
+            entry 'kotlin-stdlib-jdk7'
```

Comment:
I guess we are fine with Android, because it is for tests, but @artem-zinnatullin, please just confirm I’m overprotective.

Reply:
I would actually go with `kotlin-stdlib-jdk8`, JUnit5 requires JDK/JRE 8 anyways!

`kotlin-stdlib-jre8`

Reply:
:+1: